Source/Utility/DirectoryValueTree.cpp

```diff
@@ -0,0 +1,520 @@
+#include "DirectoryValueTree.h"
+#include "DirectoryValueTree.h"
```

Comment:
that file is _really_ included lol

Reply:
LOL! I guess I wanted to really burn those bits in! I realized I'm not sure how/if I'll apply updates to this branch per your review. But, I am working on a bug fix at the moment, so I'll apply changes there for now.